commands/volumes/volume-create.go

```diff
@@ -200,29 +204,33 @@ func volumeCreateHandler(w http.ResponseWriter, r *http.Request) {
 		Store:    "vol-create.Store",
 		Rollback: "vol-create.Rollback",
 		LogFields: &log.Fields{
-			"reqid": uuid.NewRandom().String(),
+			"reqid": reqid,
```

Comment:
My original reason for having them was that a transaction could lead to new requests on other gd2s. But that would just make things more complicated to follow. A single ID the whole way through would be better.

Doing it this way is what [opentracing](http://opentracing.io) sort of does. It allocates an ID at the beginning of a request and the ID flows through the whole cluster as actions are performed. I was looking into it to see if we could make use of it. We should try to use this sort of an established standard to trace operations instead of trying to invent our own.